### PR TITLE
Bug 810045: change dhcp client for eth0

### PR DESCRIPTION
**Pull request by dividehex**
When 'netcfg' is called to initialize and obtain an IP from dhcp it fails
on vlans with heavy dhcp traffic.  This patch changes init to call
dhcpcd instead since it handles timeouts and invalid replies better.
This also changes sut to start after dhcpcd forks and exits.

### CODE REVIEW

**Comment by michaelwu** [comment]
@tdz , want to review?

File: init.omap4pandaboard.rc [hunk 1]
```diff
@@ -90,8 +90,8 @@ service iprenew_wlan0 /system/bin/dhcpcd -n
     disabled
     oneshot
 
-service dhcpcd_eth0 /system/bin/dhcpcd -ABKL
-    disabled
+service dhcpcd_eth0 /system/bin/dhcpcd eth0
+    main class
```
Comment (tdz) [inline init.omap4pandaboard.rc]: This should be 'class main'.

File: init.omap4pandaboard.rc [hunk 2]
```diff
@@ -99,7 +99,7 @@ service iprenew_eth0 /system/bin/dhcpcd -n
     oneshot
 
 service netcfg_eth0 /system/bin/netcfg eth0 dhcp
-    class main
+    disabled
     oneshot
```
Comment (tdz) [inline init.omap4pandaboard.rc]: Please remove the whole service declaration.

**Comment by ctalbert** [comment]
@tdz Do you think you can get to this review before the weekend?  The sooner we get this reviewed and landed, the sooner we unblock release engineering.  Many many thanks.

**Comment by tdz** [comment]
Sorry. I must have missed that there was an update. I'm fine with the changes; and good work, dividehex. Sorry again.

**Pull request merged**